src/concepts/kafka-proxies/mqtt-proxy.md

```diff
@@ -0,0 +1,114 @@
+---
+description: This guide will walk through the way Zilla manages MQTT Pub/Sub connections and messages.
+---
+
+# MQTT Kafka Proxy
+
+This guide will walk through the way Zilla manages MQTT Pub/Sub connections and messages.
+
+An MQTT server acts as a broker between publishers and subscribers. This requires a complex protocol to manage the wide range of IoT devices and use cases. By proxying these messages on and off of Kafka with the [mqtt-kafka](../../reference/config/bindings/binding-mqtt-kafka.md) binding, IoT devices can transmit data to a wider range of tech stacks, adapting to more business needs.
+
+Unlike other proxies, Zilla manages the different MQTT topics instead of blindly passing them down to Kafka. This way the Kafka architecture can be optimized for MQTT Pub/Sub. MQTT client subscribers and publishers will communicate with Zilla the same as any broker.
```

Comment:
What did you mean by instead of blindly passing them down to Kafka?
Are you saying we don't map MQTT topics to Kafka topics 1-1 ?

Reply:
Yes, when i looked at waterstream and a few others it was common to map an MQTT topic to a Kafka topic. Since Zilla doesn't I wanted to call that out assuming the reader is likely to have seen those other implementations.

Reply:
I'd suggest rewording a little to mention 1-1 mapping of MQTT topics to Kafka topics, instead of "blindly".

Reply:
I agree your suggestion is a bit more clear. This page is under "concepts" and can have a more casual tone. My goal is to draw more attention to Zilla being different. Once Zilla has more routing features I will revisit this to draw a clearer parallel.